HabitLogger.gkemeza/HabitLogger.gkemeza/Methods.cs

```diff
@@ -0,0 +1,595 @@
+﻿using Microsoft.Data.Sqlite;
+using System.Globalization;
+
+namespace HabitLogger
+{
+    internal class Methods
+    {
+        Program program = new Program();
+
+        public void CreateTwoTables()
+        {
+            string connectionSource = @"Data Source=HabitLogger.db";
+
+            using (var connection = new SqliteConnection(connectionSource))
+            {
+                connection.Open();
+                var tableCommand = connection.CreateCommand();
+
+                tableCommand.CommandText = @$"
+                        CREATE TABLE IF NOT EXISTS Habits ( 
+                            id INTEGER PRIMARY KEY AUTOINCREMENT,
+                            habitName TEXT,
+                            measurementName TEXT
+                        );";
+
+                try
+                {
+                    tableCommand.ExecuteNonQuery();
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine(ex.ToString());
+                }
+            }
+
+            using (var connection = new SqliteConnection(connectionSource))
+            {
+                connection.Open();
+                var tableCommand = connection.CreateCommand();
+
+                tableCommand.CommandText = @$"
+                        CREATE TABLE IF NOT EXISTS Operations ( 
+                            id INTEGER PRIMARY KEY AUTOINCREMENT,
+                            habitID INTEGER,
+                            date DATE,
+                            measurement INTEGER
+                        );";
+
+                try
+                {
+                    tableCommand.ExecuteNonQuery();
+                }
+                catch (Exception ex)
+                {
+                    Console.WriteLine(ex.ToString());
+                }
+            }
+        }
+
+        public void GenerateHabits()
+        {
+            string habitName = "Exercise";
+            string habitMeasurement = "Squats";
+
+            string connectionSource = @"Data Source=HabitLogger.db";
+
+            using (var connection = new SqliteConnection(connectionSource))
+            {
+                connection.Open();
+                var tableCommand = connection.CreateCommand();
+
+                tableCommand.CommandText = $"INSERT INTO {program.habitsTable} (habitName, measurementName) VALUES ('{habitName}', '{habitMeasurement}');";
```

Comment:
Take a look at parameterised commands to prevent [SQL Injection](https://www.w3schools.com/sql/sql_injection.asp).